runtime/gc_vlhgc/GlobalMarkingScheme.cpp

```diff
@@ -1111,8 +1111,8 @@ MM_GlobalMarkingScheme::scanContinuationObjects(MM_EnvironmentVLHGC *env)
 						if (isMarked(object)) {
 							env->getGCEnvironment()->_continuationObjectBuffer->add(env, object);
 						} else {
-							VM_VMHelpers::cleanupContinuationObject((J9VMThread *)env->getLanguageVMThread(), object);
 							env->_markVLHGCStats._continuationCleared += 1;
+							list->cleanupContinuationObject(env, object);
```

Comment:
should be a static method and called with just a class scope. 
like this it gives a false perception that list object is relevant
admittedly, moving this method to this class is not perfect, either

Reply:
sorry for changing opinions.... but let's just move it to GCExtensions. we already touched it for the processing mode
in that case, the name of the method should still have 'Continuation'
for example releaseNativesForContinuationObject

Reply:
we already have name like scanContinuationNativeSlots in various collector Java Delegate classes.
this cleanup method is common for all collectors, hence make sense to be in GCExtensions (which is just a global Java Delegate, although it does not have 'Delegate' in its name).
overall, both place (class owner) and method name is indeed consistent with scanContinuationNativeSlots